modules/core/ImagerySystem.js

```diff
@@ -398,6 +400,99 @@ export class ImagerySystem extends AbstractSystem {
   }
 
 
+  /**
+   * getWaybackSource
+   * Resolves to an object containing the fetched Wayback imagery sources
+   * @async
+   * @returns {Promise<Object>}
+   */
+  async getWaybackSource() {
```

Comment:
This is sort of a nit too, but I've been avoiding using `async`/`await` keywords in the main Rapid code, and instead just following the naming convention that all Promise-returning functions should be named like `thingAsync`.  

I've seen people misuse `async`/`await`, and misused it myself, so I worry a bunch about writing code that's confusing about [what color it is](https://journal.stuffwithstuff.com/2015/02/01/what-color-is-your-function/).